docs/configuration.md

```diff
@@ -39,14 +39,14 @@ The `providers.github` key configures how **source{d} Lookout** will connect wit
 ```yaml
 providers:
   github:
-    comment_footer: "_If you have feedback about this comment, please, [tell us](%s)._"
+    comment_footer: "_Comment made by analyzer '{{.Name}}', [report]({{.Feedback}})._"
```

Comment:
Why not use the default value? `_If you have feedback about this comment made by the analyzer {{.Name}}, please, [tell us]({{.Feedback}})._`

Reply:
This is part of the docs; I preferred to be concise to be readable in one line; but if you really want, I can change it.